### PR TITLE
Fix some minor typos on expose-intro.html

### PR DESCRIPTION
**Pull request by JohnTitor**
Found a minor typo on expose-intro.html and fixed it.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

File: content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html [hunk 1]
```diff
@@ -66,7 +66,7 @@ <h3>Services and Labels</h3>
 		
 		<div class="row">
 			<div class="col-md-8">
-				<p>A Service routes traffic across a set of Pods. Services are the abstraction that allow pods to die and replicate in Kubernetes without impacting your application. Discovery and routing among dependent Pods (such as the frontend and backend components in an application) is handled by Kubernetes Services.</p>
+				<p>A Service routes traffic across a set of Pods. Services are the abstraction that allows pods to die and replicate in Kubernetes without impacting your application. Discovery and routing among dependent Pods (such as the frontend and backend components in an application) is handled by Kubernetes Services.</p>
```
Comment (divya-mohan0209) [inline content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html:69]: Slight nit.

```suggestion
				<p>A Service routes traffic across a set of Pods. Services are the abstraction that allows pods to die and replicate in Kubernetes without impacting your application. Discovery and routing among dependent Pods (such as the frontend and backend components in an application) are handled by Kubernetes Services.</p>
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Slight nit, please address.

File: content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html (hunk 1 above)
Comment (JohnTitor) [reply content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html]: Addressed!

**Review by divya-mohan0209 — COMMENTED** [review]
Thank you @JohnTitor 
/lgtm
/approve

**Pull request merged**